rosdep/python.yaml

```diff
@@ -3247,6 +3247,10 @@ python-pycodestyle:
   gentoo: [dev-python/pycodestyle]
   rhel: [python2-pycodestyle]
   ubuntu: [pycodestyle]
+python-pycpd:
```

Comment:
Since this is going to be pip-only key, please suffix it with `-pip`.  Also, while we are here, we may as well add in entries for debian and fedora as well.

Reply:
Thanks for the fast review. Good ideas!

Reply:
Addressed improvements suggested by @clalancette

Reply:
@clalancette Are the changes fine? I'd like to use this rosdep rule in one of my projects. ;)